SECURITY.md

```diff
@@ -0,0 +1,42 @@
+# Security policy for Flatpak
+
+ * [Supported Versions](#Supported-Versions)
+ * [Reporting a Vulnerability](#Reporting-a-Vulnerability)
+ * [Security Announcements](#Security-Announcements)
+ * [Security Vulnerability Response](#Security-Vulnerability-Response)
+
+## Supported Versions
+
+In stable branches and released packages, this table is likely to be outdated;
+please check
+[the latest version](https://github.com/flatpak/flatpak/blob/master/SECURITY.md).
+
+| Version  | Supported          | Status
+| -------- | ------------------ | -------------------------------------------------------------- |
+| 1.11.x   | :white_check_mark: | Development branch, releases may include non-security changes  |
+| 1.10.x   | :white_check_mark: | Stable branch, recommended for use in distributions            |
+| 1.9.x    | :x:                | Old development branch, no longer supported                    |
+| 1.8.x    | :white_check_mark: | Old stable branch, still supported                             |
+| <= 1.7.x | :x:                | Older branches, no longer supported                            |
+
+## Reporting a Vulnerability
+
+If you think you've identified a security issue in a Flatpak project,
```

Comment:
Should we mention something about which projects this applies to? I suppose at least Flatpak and `xdg-desktop-portal`?

Reply:
I had assumed I was initially doing this for just Flatpak itself. You're right that x-d-p also exists at a security boundary, and we should ideally add a similar document to that (but with different supported branches).

Reply:
Or x-d-p could have a shorter document that just links to this one.

x-d-p-gtk and xdg-dbus-proxy also seem security-relevant. The rest probably not?

Reply:
Makes sense, we can add security policies to those three projects separately (I can do that if you don't have time) and re-use the same mailing list. Should we change the wording here to "in Flatpak"?

I assume x-d-p-kde is covered by the KDE project's [security policy](https://community.kde.org/Policies/Security_Policy).

Reply:
- https://github.com/flatpak/xdg-desktop-portal/pull/568
- https://github.com/flatpak/xdg-desktop-portal-gtk/pull/341
- https://github.com/flatpak/xdg-dbus-proxy/pull/23